src/test/resources/features/affordability_ssttpChanges/ErrorValidation.feature

```diff
@@ -109,23 +109,55 @@ Feature: Error Validation on various pages created/updated by Affordability chan
 #      | English | 0           | -1            | 0             | monthly amount | TBC     |
 #      | English | 0           | 0             | -1            | monthly amount | TBC     |
 
-#  Scenario: Error Validation on Add Spending Page
-#    Given A user logs in and gets to the affordability pages
-#    And the user clicks continue
-#    And the user clicks on the add income link
-#    When the User toggles on English language
-#
-#    When the user enters <otherAmount> into the other income field
+  Scenario Outline: Error Validation on Add Spending Page
+    Given A user logs in and gets to the affordability pages
+    And the user clicks continue
+    And the user clicks on the add spending link
+    When the User toggles on <lang> language
+    When the user enters <Housing> into the housing field
+    When the user enters <Pension> into the pension field
+    When the user enters <Council Tax> into the council tax field
+    When the user enters <Utilities> into the utilities field
+    When the user enters <Debt> into the debt repayments field
+    When the user enters <Travel> into the travel field
+    When the user enters <Childcare> into the childcare field
+    When the user enters <Insurance> into the insurance field
+    When the user enters <Groceries> into the groceries field
+    When the user enters <Health> into the health field
+    And the user clicks continue
+    Then the <Field> field should display "<Message>"
+    Then the User toggles on English language
 
-#    And the user clicks continue
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the <Field> field should display "<Message>"
-#    Then the User toggles on English language
\ No newline at end of file
+    Examples:
+      | lang    | Housing | Pension | Council Tax | Utilities | Debt | Travel | Childcare | Insurance | Groceries | Health | Field           | Message            |
```

Comment:
This is fine - I did have a little think about this yesterday and created a single method that takes a value and puts it in each field. We can then just assert each fields error message in one test instead of all of these.

example:
- 1 enters AB!&
- 1 enters -1
- 1 enters decimal to 3 places (1.000)
Would only run the test 3 times then... instead of 30ish times here.

Happy with either, as the entire test suite is setup like this - but only mentioning as there are so many tests but with the same error messages (soon to be changed to error message + field name).

Reply:
enterAllSpendingDetails(value: String)